cypress/fixtures/unsuccessful_saving_article_response.json

```diff
@@ -0,0 +1,7 @@
+{
+  "status": 401,
+  "headers": {},
+  "body": {
+    "message": "Title cant be blank"
```

Comment:
The standard error message Rails is sending out will be "Title can't be blank"

Reply:
Solved, we just removed the 'message' field completely since we are not using it for this request

Reply:
disregard last comment, was regarding above comment. This is now fixed.